git-pr-merge

```diff
@@ -141,7 +146,7 @@ common::read_config() {
 }
 
 prmerge::parse_args() {
-  OPTSTRING=':hms'
+  OPTSTRING=':hmp:s'
```

Comment:
I tried to understand why there has been a colon added but couldn't quite.

Reply:
the opt string passed to `getopts` parses this. If a flag has a colon after it, it means that flag takes an argument. The colon at the beginning is special and means something completely different - that stops getopts reporting errors so we can do it ourself.

On bash, you can run `help getopts` to see docs for this. Not sure if zsh does that or not.